app/bundles/CoreBundle/Views/Exception/base.html.twig

```diff
@@ -0,0 +1,146 @@
+                    <h2><i class="fa fa-quote-left"></i> <strong>{{ status_code }}</strong> {{ status_text }}{{ exceptionMessage }} <i class="fa fa-quote-right"></i></h2>
+                    <footer class="text-right">Mautibot</footer>
+                </blockquote>
+                <div class="pull-right">
+                    <a class="text-muted" href="http://mau.tc/report-issue" target="_new">{{ 'mautic.core.report_issue' | trans }}</a>
+                </div>
+            </div>
+            {% endif %}
+        </div>
+
+        {% if isInline %}
+        <div class="slimscroll" style="max-height: 200px;">
+        {% else %}
+        <div>
+        {% endif %}
+
+            <div class="row mt-20">
+                <div class="col-sm-12">
+                    <h5 class="ml-lg text-danger">{{ exception.class }}</h5>
+                    <div class="well well-sm ma-md">
+                        {{ include ('MauticCoreBundle:Exception:traces.html.twig', {
+                            'traces': exception.trace,
+                        })}}
+                    </div>
+                </div>
+            </div>
+
+            {% if previousExceptions and previousExceptions|length > 0 %}
+            <div class="row mt-20">
+                <div class="col-sm-12">
+                    <h5 class="ml-lg">{{ 'mautic.core.error.previousexceptions' | trans }}</h5>
+                    <div class="panel-group" id="previous" role="tablist" aria-multiselectable="true">
+                    {% for key, e  in previousExceptions %}
+                        <div class="panel panel-default">
+                            <div class="panel-heading" role="tab" id="previous_heading_{{ key }}">
+                                <h4 class="panel-title pa-sm">
+                                    <a data-toggle="collapse" data-parent="#previous" href="#previous_body_{{ key }}" aria-expanded="true" aria-controls="collapseOne">
+                                        {{ e.message }}
+                                    </a>
+                                </h4>
+                            </div>
+                            <div id="previous_body_{{ key }}" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne">
+                                <div class="panel-body">
+                                    <div class="pa-sm">
+                                        {{ include ('MauticCoreBundle:Exception:traces.html.twig', {
+                                            'traces': e.trace,
+                                        })}}
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    {% endfor %}
+                    </div>
+                </div>
+            {% endif %}
+            </div>
+            
+        
+        </div>
+    </div>
+    <div class="clearfix"></div>
+
+    <pre>
+        {{ dump(exception) }}
```

Comment:
`dump` isn't available in production environment.

Reply:
Yeah, had a brain fart there :D luckily the linter also caught the issue.